compound_disease/compound_treats_disease/edge_prediction_experiment/edge_prediction_experiment.py

```diff
@@ -0,0 +1,330 @@
+
+# coding: utf-8
+
+# # Compound Treats Disease Edge Prediction
+
+# This notebook is designed to take the next step moving from predicted sentences to edge predictions
```

Comment:
I would maybe be a bit more specific about what you mean by moving from predicting sentences to predicting edges